### PR TITLE
feature(Chromecast): add a new a get orientation method

### PR DESCRIPTION
**Pull request by snooky23**

### CODE REVIEW

File: ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h [hunk 1]
```diff
@@ -19,7 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)showExtededPlayer;
 - (nonnull UIViewController *)getExtededPlayerViewController;
 - (void)setCastDelegate:(nonnull id)castDelegate;
-
```
Comment (guykogus) [inline ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Don't delete the empty line. White spacing = beauty :)

Comment (snooky23) [reply ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Done. please approve.

**Comment by liviur** [comment]
In terms of functionality is it different than the initial orientation as I created here:
https://github.com/applicaster/ZappAppConnector-iOS/blob/master/ZappAppConnector/Protocols/ZAAppDelegateConnectorLayoutsStylesProtocol.h
?

File: ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h [hunk 2]
```diff
@@ -19,6 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)showExtededPlayer;
 - (nonnull UIViewController *)getExtededPlayerViewController;
 - (void)setCastDelegate:(nonnull id)castDelegate;
+- (NSUInteger)getExtededPlayerOrientation;
```
Comment (guykogus) [inline ZappAppConnector/Protocols/ZAAppDelegateConnectorChromecastProtocol.h]: Should be `getExtendedPlayerOrientation`, no?

**Comment by snooky23** [comment]
@liviur - This is for knowing what is the UIInterfaceOrientationMask of the expended Chromecast controls. in the future, it can have a different orientation functionality from the app.

**Pull request merged**